src/components/DocumentationTopic/TopicsLinkCardGrid.vue

```diff
@@ -9,43 +9,100 @@
 -->
 
 <template>
-  <div class="TopicsLinkCardGrid">
-    <Row :columns="{
-      large: compactCards ? 3 : 2,
-      medium: 2,
-    }">
-      <Column
-        v-for="item in items"
-        :key="item.title"
-      >
-        <TopicsLinkCardGridItem :item="item" :compact="compactCards" />
-      </Column>
-    </Row>
-  </div>
+  <Pager
+    :aria-label="$t('links-grid.label')"
+    :class="['TopicsLinkCardGrid', topicStyle]"
+    :pages="pages"
+  >
+    <template #page="{ page }">
+      <Row :columns="{
+        large: compactCards ? 3 : 2,
+        medium: 2,
+      }">
+        <Column
+          v-for="item in page"
+          :key="item.title"
+        >
+          <TopicsLinkCardGridItem :item="item" :compact="compactCards" />
+        </Column>
+      </Row>
+    </template>
+    <BreakpointEmitter @change="handleBreakpointChange" />
+  </Pager>
 </template>
 
 <script>
-import Row from 'docc-render/components/ContentNode/Row.vue';
+import BreakpointEmitter from 'docc-render/components/BreakpointEmitter.vue';
 import Column from 'docc-render/components/ContentNode/Column.vue';
+import Pager from 'docc-render/components/Pager.vue';
+import Row from 'docc-render/components/ContentNode/Row.vue';
 import { TopicSectionsStyle } from 'docc-render/constants/TopicSectionsStyle';
+import { BreakpointName } from 'docc-render/utils/breakpoints';
+import { page } from 'docc-render/utils/arrays';
 import TopicsLinkCardGridItem from './TopicsLinkCardGridItem.vue';
 
 export default {
   name: 'TopicsLinkCardGrid',
-  components: { TopicsLinkCardGridItem, Column, Row },
+  components: {
+    BreakpointEmitter,
+    Column,
+    Pager,
+    Row,
+    TopicsLinkCardGridItem,
+  },
+  data: () => ({
+    breakpoint: BreakpointName.large,
+  }),
   props: {
     items: {
       type: Array,
       required: true,
     },
+    pageSize: {
+      type: Number,
+      required: false,
+    },
     topicStyle: {
       type: String,
       default: TopicSectionsStyle.compactGrid,
       validator: v => v === TopicSectionsStyle.compactGrid || v === TopicSectionsStyle.detailedGrid,
     },
+    usePager: {
+      type: Boolean,
+      default: false,
+    },
   },
   computed: {
     compactCards: ({ topicStyle }) => topicStyle === TopicSectionsStyle.compactGrid,
+    defaultPageSize: ({
+      breakpoint,
+      items,
+      topicStyle,
+      usePager,
+    }) => (usePager ? {
+      [TopicSectionsStyle.compactGrid]: {
+        [BreakpointName.large]: 6,
+        [BreakpointName.medium]: 4,
+        [BreakpointName.small]: 1,
+      },
+      [TopicSectionsStyle.detailedGrid]: {
+        [BreakpointName.large]: 4,
+        [BreakpointName.medium]: 2,
+        [BreakpointName.small]: 1,
+      },
```

Comment:
Could we add more detailed documentation here?
For example, `compactGrid` large breakpoint is 2 row by 3 columns; This information is already available, but needs to do a bit math to figure it out.